syntax-color-doc/syntax-color/syntax-color.scrbl

```diff
@@ -197,46 +258,76 @@ Like @racket[racket-lexer], but with several differences:
 
 ]}
 
+@defproc[(module-lexer* [in input-port?]
+                        [offset exact-nonnegative-integer?]
+                        [mode (or/c #f
+                                    (-> input-port? any)
+                                    (cons/c (-> input-port? any/c any) any/c))])
+         (values (or/c string? eof-object?) 
+                 (or/c symbol?
+                       (and/c (hash/c symbol? any/c) immutable?))
+                 (or/c symbol? #f) 
+                 (or/c number? #f) 
+                 (or/c number? #f)
+                 exact-nonnegative-integer?
+                 (or/c #f 
+                       (-> input-port? any)
+                       (cons/c (-> input-port? any/c any) any/c)))]{
+
+Like @racket[module-lexer*], except that the attribute result
```

Comment:
@mflatt do you mean `Like @racket[racket-lexer*]`?

Reply:
Yes, thanks!